theme/palette/elements.less

```diff
@@ -7,3 +7,4 @@
 @sk-footer: @sk-mid-gray-light;
 @sk-background: @sk-gray-light;
 @sk-yellow: #fffce3;
+@sk-gold: #ffc600;
```

Comment:
I recommend that @sk-yellow is #FFC600, as our primary yellow, and #FFFCE3 should be @sk-yellow-light, since that is the naming pattern we've used for the greens.